SIG: initial commit for planner sig (#96)
pingcap/community: special-interest-groups/sig-planner/README.md

# Planner Special Interest Group(Planner SIG)

Planner SIG covers TiDB planner/statistics implementations. We are focused on
the development and maintenance of the TiDB [planner](https://pingcap.com/docs/stable/reference/performance/sql-optimizer-overview/)

> and [statistics](https://pingcap.com/docs/stable/reference/performance/statistics/) component, including but not limited to:

Reviewer: ```suggestion
and [statistics](https://pingcap.com/docs/stable/reference/performance/statistics/) components, including but not limited to:
```

Reviewer: You can also describe the mission of TiDB Planner

* Supporting new features.
* Improving the performance.
* Fixing bugs.

## Roles and Organization Management

See [Planner SIG Roles and Organization Management](./roles-and-organization-management.md)

## Members

See [SIG Planner Member List](./member-list.md)

## Meetings

> * Regular Working Group Meeting: 5:00, Thursday at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).

Reviewer 2: 5:00 AM or 5:00 PM?

Author: AM. can be checked by that link

* Meeting Zoom: [ttps://pingcap.zoom.com.cn/j/492702291](ttps://pingcap.zoom.com.cn/j/492702291)

> * Meeting Notes: to be updated

Reviewer: please add a public meeting notes doc.

Author: Do we need to fill it before first meeting is held?

Reviewer 2: I think a meeting note link is ok, even there is no content for now.

## Contact

> * Slack: channel #sig-planner in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.

Reviewer 2: ```suggestion
* Slack: channel #sig-planner in the [tidbcommunity](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) slack workspace.
```

## Code Locations

* The [planner package](https://github.com/pingcap/tidb/tree/master/planner) in the tidb repo.
* The [statistics package](https://github.com/pingcap/tidb/tree/master/statistics) in the tidb repo.

> * Other codes that affect the correctness, performance of the planner, like say the [expresion package](https://github.com/pingcap/tidb/tree/master/expression).

Reviewer 2: ```suggestion
* Other codes that affect the correctness, performance of the planner, like say the [expression package](https://github.com/pingcap/tidb/tree/master/expression).
```
